client/jetpack-connect/user-type/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from 'components/formatted-header';
+import jetpackOnly from '../jetpack-only';
+import MainWrapper from '../main-wrapper';
+import UserTypeForm from './form';
+import withTrackingTool from 'lib/analytics/with-tracking-tool';
+import WpcomColophon from 'components/wpcom-colophon';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { saveSiteUserType } from 'state/jetpack-connect/actions';
+
+class JetpackUserType extends Component {
+	handleSubmit = userType => {
+		const { siteId, siteSlug } = this.props;
+
+		this.props.saveSiteUserType( siteId, userType );
+
+		page.redirect( `/jetpack/connect/plans/${ siteSlug }` );
```

Comment:
This seems to haven't ben addressed either - we need to change this to `page.` - see https://github.com/Automattic/wp-calypso/pull/31297#discussion_r265560475